src/VisualStudio/CSharp/Impl/PackageRegistration.pkgdef

```diff
@@ -18,8 +18,16 @@
 @="{694dd9b6-b865-4c5b-ad85-86356e9c88dc}"
 "Package"="{13c3bbb4-f18f-4111-9f54-a0fb010d9194}"
 "LangResID"=dword:00000065
+
+"DefaultToInsertSpaces"=dword:00000001
+"EnableAdvancedMembersOption"=dword:00000001
 "RequestStockColors"=dword:00000001
+"ShowBraceCompletion"=dword:00000001
+"ShowCompletion"=dword:00000001
 "ShowDropdownBarOption"=dword:00000001
+"ShowSmartIndent"=dword:00000001
+"SupportsImageCaching"=dword:00000001
```

Comment:
do we need to make all these other changes?

Reply:
I was very confused when starting this PR why C# was working in the first place, the answer being there were settings being set in the VS repo directly. So whereas this isn't necessary, it's trying to make the world a bit saner of a place.

Reply:
I like sanity